_sass/blocks/explore/_filters.scss

```diff
@@ -76,7 +76,17 @@
 // }
 
 .unit-selector {
+  display: inline-block;
+  background-color: $white;
+  @include appearance(none);
```

Comment:
Expected item on line 81 to appear before line 79. Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets